deployment-repository/src/main/java/org/jboss/as/repository/ContentRepository.java

```diff
@@ -53,7 +58,7 @@
  * @author John Bailey
  */
 public interface ContentRepository {
-
+    String UNSUPPORTED_PROPERTY = "org.wildfly.unsupported.content.repository.obscolescence";
```

Comment:
I'd rather hard code the string in multiple places than make this part of this interface.